osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -814,6 +821,9 @@ private void updateScrollPosition()
 
                         Scroll.ScrollTo(scrollTarget.Value, false);
 
+                        foreach (var i in root.Children)
```

Comment:
this is still updating the y position of every item, right..?

Reply:
Yes, but this is immediate scroll only. I'll have to check, but isn't immediate scroll done only on filter changes?

Reply:
Looks to be the case for now. At very least, this needs an inline comment mentioning this is the case (because it's not obvious when reading through the code).